speakeasy/windows/objman.py

```diff
@@ -501,12 +501,12 @@ class Process(KernelObject):
     """
     An EPROCESS object used by the Windows kernel to represent a process
     """
-    ldr_entries = []
 
     def __init__(self, emu, pe=None, user_modules=[],
                  name='', path='', cmdline='', base=0, session=0):
         super(Process, self).__init__(emu=emu)
         # TODO: For now just allocate a blank opaque struct for an EPROCESS
+        ldr_entries = []
```

Comment:
i think this at least has to become `self.ldr_entries`.

i cannot comment on whether the semantics of this work or not.

Reply:
yeah, you are right, the correct way is `self.ldr_entries`